src/data/actions/utils.js

```diff
@@ -25,39 +26,65 @@ const sortAlphaAsc = (gradeRowA, gradeRowB) => {
 
 const sortNumerically = (colKey, direction) => {
   function sortNumAsc(gradeRowA, gradeRowB) {
-    if (gradeRowA[colKey] < gradeRowB[colKey]) {
+    let a;
```

Comment:
Can you make sure you name the variables what they are? I don't like the meaningless variable names unless they are absolutely meaningless

Reply:
its a common pattern for sorting. We don't actually know what a or b is. It all depends on what is passed in

Reply:
I can call it `valueAtGradeRowA` but I feel like that is just as arbitrary as `a`